stellar_dew.iml

```diff
@@ -4,25 +4,8 @@
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
-      <sourceFolder url="file://$MODULE_DIR$/test" isTestSource="true" />
```

Comment:
Why is this stuff deleted? Is the folder for the test not right here?

Reply:
I don't know, I didn't deleted it manually.